src/Repository/PostRepository.php

```diff
@@ -172,12 +175,16 @@ private function findNextOrPreviousPost(int $id, int $direction = 1): ?Post
     }
 
     /**
-     * @throws \Exception
+     * @throws Exception
      */
     public function filterOutdatedEventscallback(Post $post): bool
     {
-        $validPost = ($post->getEndDate() >= (new \DateTime('now')));
+        if ($post->getEndDate()) {
+            $isValidPost = ($post->getEndDate() >= (new \DateTime('now')));
```

Comment:
Maybe you should also check for endDate > startDate

Reply:
Isn't it pointless as EventPostType form builder constraints makes it impossible to record end date before start date?

Reply:
Right, nevermind! :+1:  (Didn't understand this block)